src/app/app.component.css

```
@@ -0,0 +1,37 @@
+.container {
+  margin: 30px auto;
+  width: 50%;
+  text-align: center;
+}
+
+.inpWrapper {
```

Що таке `inpWrepper`? 
```suggestion
.input-wrapper {
```

---

если не против Кемел Кейсом мне проще...привычней  inputWrapper